### DISCUSSION TITLE
Maliit support

### DISCUSSION THREAD

**Discussion by chandradeepdey** [OpenBangla/OpenBangla-Keyboard#245] [Ideas]
Yet another input method framework. This one is for virtual keyboards. Important for phone distributions, such as those using Plasma Mobile, and touch screen displays in general.

https://maliit.github.io/

**Comment by mominul** [comment]
Wow, does it support Android?

**Reply by chandradeepdey** [reply]
No unfortunately. Android's IME system is probably very different from Wayland's protocols.

**Reply by chandradeepdey** [reply]
Other than Plasma Mobile, Maliit is used by default on Ubuntu Touch afaik.
